libs/medic-users-meta/migrations/202105171933.do.86.usersMeta.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE IF NOT EXISTS couchdb_users_meta (doc jsonb);
+
+CREATE MATERIALIZED VIEW useview_feedback AS
+SELECT
+    doc->>'_id' AS uuid,
+    doc#>>'{meta,source}' AS SOURCE,
+    doc#>>'{meta,url}' AS url,
+    doc#>>'{meta,user,name}' AS user_name,
+    doc#>>'{meta,time}' AS period_start,
+    COALESCE(doc#>>'{info,cause}',doc->>'info') AS cause,
+    doc#>>'{info,message}' AS message
+FROM
+    couchdb_users_meta
+WHERE
+    doc->>'type'='feedback';
+
+CREATE UNIQUE INDEX idx_useview_feedback_uuid ON useview_feedback(uuid);  --> Only to allow the refresh of the view CONCURRENTLY
```

Comment:
According to this thread, app_services opted to keep the old indexes, without the `unique` constraint for querying purposes: https://medic.slack.com/archives/C0813CQRX/p1621248880109500?thread_ts=1621245953.105300&cid=C0813CQRX

Reply:
Right, added again without the unique constraint.